.github/workflows/ci.yaml

```diff
@@ -712,7 +712,7 @@ jobs:
           just test
 
   benchmarks:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
We had some issues in the past where GitHub upgrade and downgraded us between versions which broke CI (because of caching). Could you fix the runner to whatever latest resolves today (24?)

Reply:
Yes